public/image_process.py

```diff
@@ -29,6 +29,8 @@ async def upload_single_image(origin_image, file_name):
 
 
 async def upload_file():
+    from js import imageUpload
+
```

Comment:
Why need to move import here?

Reply:
@pkong-ds added comment in script
> Since we will update `imageUpload` when calling this function,
> need to re-import it to force update to new value
> or we will always generate first uploaded image

Reply:
> Since we will update imageUpload when calling this function,
😱 which line `update imageUpload`? I view a bit, couldn't see